libr/include/r_endian.h

```diff
@@ -0,0 +1,211 @@
+#ifndef R_ENDIAN_H
+#define R_ENDIAN_H
+
+/* Endian agnostic functions working on single byte. */
+
+static inline ut8 read_ble8(const void *src) {
```

Comment:
pls discuss proper naming before changing it all the way

> On 03 May 2016, at 13:18, Damien Zammit <notifications@github.com> wrote:
> 
> In libr/include/r_endian.h <https://github.com/radare/radare2/pull/4715#discussion_r61866485>:
> 
> > @@ -0,0 +1,211 @@
> > +#ifndef R_ENDIAN_H
> > +#define R_ENDIAN_H
> > +
> > +/* Endian agnostic functions working on single byte. */
> > +
> > +static inline ut8 read_ble8(const void *src) {
> Ok
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/radare/radare2/pull/4715/files/61bb6d83544b22ac8e02d42ea5a581b3d713e86e#r61866485>